modules/nw-configure-sysctl-interface-sriov-network.adoc

```diff
@@ -0,0 +1,171 @@
+// Module included in the following assemblies:
+//
+//networking/hardware_networks/configuring-sriov-device.adoc
+
+:_content-type: PROCEDURE
+[id="configure-sysctl-on-sriov-network_{context}"]
+= Configure sysctl on a SR-IOV network 
+
+As a cluster administrator, set the interface specific sysctls on virtual interfaces created by SR-IOV by adding the tuning configuration to the optional `metaPlugins` parameter of the `SriovNetwork` resource.
+
+The SR-IOV Network Operator manages additional network definitions. When you specify an additional SR-IOV network to create, the SR-IOV Network Operator creates the `NetworkAttachmentDefinition` custom resource (CR) automatically.
+
+[NOTE]
+====
+Do not edit `NetworkAttachmentDefinition` custom resources that the SR-IOV Network Operator manages. Doing so might disrupt network traffic on your additional network.
+====
+
+To change the interface-level network `net.ipv4.conf.IFNAME.accept_redirects` sysctl create an additional SR-IOV network attachment with the Container Network Interface (CNI) tuning plug-in by using the following procedure.
+
+.Prerequisites
+
+* Install the {product-title} CLI (oc).
+* Log in to the {product-title} cluster as a user with cluster-admin privileges.
+
+.Procedure
+
+. Create the `SriovNetwork` custom resource (CR) for the additional SR-IOV network attachment and insert the `metaPlugins` configuration, as in the following example CR. Save the YAML as the file `sriov-network-interface-sysctl.yaml`.
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: onevalidflag <1>
+  namespace: openshift-sriov-network-operator <2>
+spec:
+  resourceName: policy-example-oneflag <3>
+  networkNamespace: sysctl-tuning-test <4>
```

Comment:
Some of the example resources are located in default namespace, some of them in demo namespace and this particular on in sysctl-tuning-test. I think we should use the same namespace for our examples